### PR TITLE
switch to internal images for cnf-test

### PR DESCRIPTION
**Pull request by SchSeba**
Signed-off-by: Sebastian Sch <sebassch@gmail.com>

### CODE REVIEW

File: ci-operator/config/openshift-kni/cnf-features-deploy/openshift-kni-cnf-features-deploy-master.yaml [hunk 1]
```diff
@@ -3,14 +3,18 @@ base_images:
     name: "4.9"
     namespace: ocp
     tag: base
+  ocp_builder_rhel-8-golang-1.15-openshift-4.7:
```
Comment (yuvalk) [inline ci-operator/config/openshift-kni/cnf-features-deploy/openshift-kni-cnf-features-deploy-master.yaml:6]: 4.7?
I was able to find one without ocp affinity:
`registry.svc.ci.openshift.org/openshift/release:golang-1.13`
maybe there's one for 1.15, 1.16 too

File: ci-operator/config/openshift-kni/cnf-features-deploy/openshift-kni-cnf-features-deploy-master.yaml [hunk 2]
```diff
@@ -83,7 +91,7 @@ tests:
     cluster_profile: gcp
     test:
     - as: e2e-gcp-ovn-periodic
-      commands: make functests-on-ci
+      commands: IPERF3_BITRATE_OVERRIDE="300000000" make functests-on-ci
```
Comment (yuvalk) [inline ci-operator/config/openshift-kni/cnf-features-deploy/openshift-kni-cnf-features-deploy-master.yaml:94]: how is that related to internal images?
probably need a separate commit

File: ci-operator/config/openshift-kni/cnf-features-deploy/openshift-kni-cnf-features-deploy-master.yaml [hunk 3]
```diff
@@ -96,7 +104,7 @@ tests:
     cluster_profile: gcp
     test:
     - as: e2e-gcp-periodic
-      commands: make functests-on-ci
+      commands: SKIP_TESTS="ovs_qos" make functests-on-ci
```
Comment (yuvalk) [inline ci-operator/config/openshift-kni/cnf-features-deploy/openshift-kni-cnf-features-deploy-master.yaml:107]: separate commit?

File: ci-operator/config/openshift-kni/cnf-features-deploy/openshift-kni-cnf-features-deploy-master.yaml (hunk 2 above)
Comment (yuvalk) [reply ci-operator/config/openshift-kni/cnf-features-deploy/openshift-kni-cnf-features-deploy-master.yaml:94]: plz also explain why is this needed at all
and wont it be smarter to have it in the git repo and not openshift/release?

File: ci-operator/config/openshift-kni/cnf-features-deploy/openshift-kni-cnf-features-deploy-master.yaml (hunk 1 above)
Comment (SchSeba) [reply ci-operator/config/openshift-kni/cnf-features-deploy/openshift-kni-cnf-features-deploy-master.yaml:6]: I am using the same config as all the other operators for example:
sriov - https://github.com/openshift/release/blob/master/ci-operator/config/openshift/sriov-network-operator/openshift-sriov-network-operator-master.yaml
ptp - https://github.com/openshift/release/blob/master/ci-operator/config/openshift/ptp-operator/openshift-ptp-operator-master.yaml

we can open a follow up PR to bump the version in the repo and then add new images here as for example the machine-config-operator

https://github.com/openshift/release/blob/master/ci-operator/config/openshift/machine-config-operator/openshift-machine-config-operator-master.yaml#L49

**Comment by fromanirh** [comment]
what's the status? can we help to move this forward?

**Comment by yuvalk** [comment]
/override ci/rehearse/periodic-ci-openshift-kni-cnf-features-deploy-master-e2e-bastion-periodic
it is not working ATM, we are separately working on fixing it.

File: ci-operator/config/openshift-kni/cnf-features-deploy/openshift-kni-cnf-features-deploy-master.yaml (hunk 1 above)
Comment (cynepco3hahue) [reply ci-operator/config/openshift-kni/cnf-features-deploy/openshift-kni-cnf-features-deploy-master.yaml:6]: IMHO we should use the right version rhel-8-golang-1.16-openshift-4.9 from the beginning.

**Comment by yuvalk** [comment]
> /retest

we need to figure a way to remove (or override) the ci/rehearse/periodic-ci-openshift-kni-cnf-features-deploy-master-e2e-bastion-periodic

**Comment by fromanirh** [comment]
> > /retest
> 
> we need to figure a way to remove (or override) the ci/rehearse/periodic-ci-openshift-kni-cnf-features-deploy-master-e2e-bastion-periodic

I think this could require a rebase + re run `make ci-job` + a reupload

**Review by cynepco3hahue — COMMENTED** [review]
> we need to figure a way to remove (or override) the ci/rehearse/periodic-ci-openshift-kni-cnf-features-deploy-master-e2e-bastion-periodic

It is not the required one, so it will not prevent the merge.

File: ci-operator/config/openshift-kni/cnf-features-deploy/openshift-kni-cnf-features-deploy-master.yaml (hunk 1 above)
Comment (cynepco3hahue) [reply ci-operator/config/openshift-kni/cnf-features-deploy/openshift-kni-cnf-features-deploy-master.yaml:6]: And the same is true for SR-IOV and PTP operators.

Comment (fromanirh) [reply ci-operator/config/openshift-kni/cnf-features-deploy/openshift-kni-cnf-features-deploy-master.yaml:6]: If we do a re-upload, I agree we should change.

**Comment by fromanirh** [comment]
CI lane failure: `verify-commits` fail
images lane failure: throttled by docker pulls :)
I expect cascading failures

File: ci-operator/config/openshift-kni/cnf-features-deploy/openshift-kni-cnf-features-deploy-master.yaml (hunk 1 above)
Comment (SchSeba) [reply ci-operator/config/openshift-kni/cnf-features-deploy/openshift-kni-cnf-features-deploy-master.yaml]: we also change the images on cnf-feature-deploy so we must update :)

Comment (cynepco3hahue) [reply ci-operator/config/openshift-kni/cnf-features-deploy/openshift-kni-cnf-features-deploy-master.yaml]: Thanks!

**Comment by yuvalk** [comment]
> CI lane failure: `verify-commits` fail
fixed!

> images lane failure: throttled by docker pulls :)
wasnt this pr ment to erradicate docker from here?

**Comment by SchSeba** [comment]
> images lane failure: throttled by docker pulls :)
> wasnt this pr ment to erradicate docker from here?

This was fixed I had to bump the switch to 1.16 as we update the dockerfile 9 days ago

**Comment by yuvalk** [comment]
> /retest

it passed all required tests
rehearsals are not required!
